specification/ai/Azure.AI.Agents/run_steps/models.tsp

```diff
@@ -381,17 +381,28 @@ model RunStepDeltaBingGroundingToolCall extends RunStepDeltaToolCall {
   bingGrounding: Record<string>;
 }
 
-@doc("Represents the Custom Bing Grounding tool call in a streaming run step.")
+@doc("Represents the Bing Custom Search tool call in a streaming run step.")
 @added(Versions.v2025_05_15_preview)
 model RunStepDeltaCustomBingGroundingToolCall extends RunStepDeltaToolCall {
   @doc("The object type, which is always 'bing_custom_search'.")
   type: "bing_custom_search";
 
-  @doc("The dictionary with request and response from Custom Bing Grounding search tool.")
+  @doc("The dictionary with request and response from Bing Custom Search tool.")
   @encodedName("application/json", "bing_custom_search")
   bingCustomSearch: Record<string>;
 }
 
+@doc("Represents the Azure Function tool call in a streaming run step.")
+@added(Versions.v1)
+model RunStepDeltaAzureFunctionToolCall extends RunStepDeltaToolCall {
+  @doc("The object type, which is always 'azure_function'.")
+  type: "azure_function";
+
+  @doc("Partial description of an Azure function call.")
+  @encodedName("application/json", "azure_function")
```

Comment:
This is a tangent for this PR in particular, but does it cause problems for us to use the snake-cased identifier as the variable name in the TypeSpec directly (that is, without use of `@encodedName`)?

It *used to be* that it'd generate a warning/error for non-standard naming, but I believe that's been relaxed for quite some time and specs will now accept that directly. Generators *should* meanwhile adjust the names accordingly since it's still a standard naming pattern.

Not important to address here (better done all at once) -- just raising it as an optimization note.

<img width="1509" height="335" alt="image" src="https://github.com/user-attachments/assets/d6de23ee-81c0-474c-9be8-0289a5551f59" />

Reply:
Fixed.